docs/zh/command/cinfo.md

```
@@ -18,11 +18,23 @@ cinfo
   - up: 可用
   - down: 不可用
 - **NODES**：节点数
-- **STATE**： 节点状态
-  - **idle**： 空闲
-  - mix： 节点部分核心可以使用
-  - alloc： 节点已被占用
-  - down： 节点不可用
+- **STATE**： 节点状态，包括资源状态和控制状态
+  -   **资源状态**
+    -     idle： 空闲
```

markdownlint-cli2 (0.21.0)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 4

(MD005, list-indent)